WordPress/UITests/Tests/MainNavigationTests.swift

```diff
@@ -8,7 +8,7 @@ class MainNavigationTests: XCTestCase {
         setUpTestSuite()
 
         try LoginFlow.login(siteUrl: WPUITestCredentials.testWPcomSiteAddress, email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
-        mySiteScreen = try TabNavComponent()
```

Comment:
With the test no longer using the stored `mySiteScreen`, we should be able to remove the property from the class, too. See line 5 of this file.